core/internal/testutils/testutils.go

```diff
@@ -337,8 +336,8 @@ func RequireLogMessage(t *testing.T, observedLogs *observer.ObservedLogs, msg st
 //
 // Get a *observer.ObservedLogs like so:
 //
-// 		observedZapCore, observedLogs := observer.New(zap.DebugLevel)
-// 		lggr := logger.TestLogger(t, observedZapCore)
+//	observedZapCore, observedLogs := observer.New(zap.DebugLevel)
+//	lggr := logger.TestLogger(t, observedZapCore)
```

Comment:
I have go 1.19rc1 installed, so I guess go fmt does this now. Probably related to the docs changes.